moped-editor/src/queries/tableLookups.js

```diff
@@ -67,9 +67,8 @@ export const TABLE_LOOKUPS_QUERY = gql`
       order_by: { type: asc, name: asc }
     ) {
       id
-      type
-      name
       slug
+      full_name
```

Comment:
The `type` and `name` columns are still rendering in the data dictionary table. Looks like you just update the settings to use this new column.

<img width="682" alt="Screenshot 2025-05-21 at 10 18 46 AM" src="https://github.com/user-attachments/assets/00cd89aa-23f5-4caa-b168-70e62c88a4d4" />